### PR TITLE
Order CEs by MPS date

### PR DESCRIPTION
**Pull request by tej99**
Collection exercises need to be ordered by MPS date until standardisation of period.

https://trello.com/c/CDOioVUr/84-us111-display-collection-exercise-periods-in-order

### CODE REVIEW

**Comment by tej99** [comment]
[US111: Display Collection Exercise Periods in chronological order of MPS date](https://trello.com/c/CDOioVUr/84-us111-display-collection-exercise-periods-in-order)

**Comment by tej99** [comment]
@emilioward I find collection_exercise quite long for a var name, and because of that the line exceeds max length and the line breaks I have to put in makes the code difficult to read.

**Comment by JamesGardiner** [comment]
There should at least be a test for the new sorting logic

File: tests/views/test_survey.py [hunk 1]
```diff
@@ -9,26 +9,27 @@
...
 
 url_get_survey_list = f'{app.config["BACKSTAGE_API_URL"]}/v1/survey/surveys'
 url_get_legal_basis_list = f'{app.config["SURVEY_URL"]}/legal-bases'
 url_create_survey = f'{app.config["SURVEY_URL"]}/surveys'
 
-with open('tests/test_data/survey/survey_list.json') as json_data:
-    survey_list = json.load(json_data)
-with open('tests/test_data/survey/legal_basis_list.json') as json_data:
-    legal_basis_list = json.load(json_data)
+with open('tests/test_data/survey/survey_list.json') as f:
```
Comment (rhiGriff) [inline tests/views/test_survey.py]: These should be as fp as @JamesGardiner mentioned in this PR: https://github.com/ONSdigital/ras-frontstage/pull/328#discussion-diff-187295508R27

**Pull request merged**